docker-entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+if [ ! -f /etc/ld-relay.conf ]; then
+# only create /etc/ld-relay.conf if it doesn't exist already
+
+if ! env | grep -q LD_ENV_ ; then
+  echo "WARNING: at least one LD_ENV_ should be set" >&2
+fi
+
+echo "
+[main]
+streamUri = \"${STREAM_URI:-https://stream.launchdarkly.com}\"
+baseUri = \"${BASE_URI:-https://app.launchdarkly.com}\"
+exitOnError = ${EXIT_ON_ERROR:-false}
+port = ${PORT:-8030}
```

Comment:
The Dockerfile exposes 8030, yet here the port is configurable. consider simply leaving this non-configurable as you can map the port with docker anyways. Also, `PORT` is a generic Env name and can clash with existing Vars - consider a more unique name like LD_RELAY_PORT.